pcsx2/System.cpp

```diff
@@ -211,12 +211,21 @@ void SysLogMachineCaps()
 	Console.Newline();
 
 	Console.WriteLn( Color_StrongBlack, "Host Machine Init:" );
-
+	
+	#ifdef __unix__
+	Console.Indent().WriteLn(
+		L"Operating System =  %s",
+    	WX_STR(wxGetLinuxDistributionInfo().Description)
+    );
+	#endif
+	#ifdef __WIN32__
+	Console.Indent().WriteLn(
+		L"Operating System =  %s\n",
+		WX_STR(GetOSVersionString())
+	);
+	#endif
```

Comment:
going to take a guess and say this breaks macos/bsd.
Don't you think something like this might be clearer?
```cpp
wxString os_string;
#ifdef __linux__
os_string = wxGetLinuxDistributionInfo().Description;
#else
os_string = GetOSVersionString();
#endif

Console.Indent().WriteLn(
	L"Operating System =  %s\n"
	L"Physical RAM     =  %u MB",
	os_string,
	(u32)(GetPhysicalMemory() / _1mb)
);
```

Reply:
better yet maybe just modify this function?
https://github.com/PCSX2/pcsx2/blob/6f7890b709d5e3f7f5b824781e493455efc92339/common/src/Utilities/Linux/LnxMisc.cpp#L53-L56